packages/api/package.json

```diff
@@ -26,8 +26,11 @@
     "http-errors": "^1.8.0",
     "lambda-multipart-parser": "^1.0.1",
     "middy-middleware-json-error-handler": "^3.0.0",
-    "serialize-error": "^8.1.0",
```

Comment:
Was this deleted intentionally?

Reply:
No, looks like this is due to reordering.  This same package is listed on line 32.

Reply:
Sorry my mistake, I must have over looked it.